packages/turbo/android/src/main/java/com/reactnativeturbowebview/RNVisitableView.kt

```diff
@@ -172,14 +172,17 @@ class RNVisitableView(context: Context) : LinearLayout(context), SessionSubscrib
 
     turboView.attachWebView(webView) { attachedToNewDestination ->
       onReady(attachedToNewDestination)
+      sendEvent(RNVisitableViewEvent.WEB_VIEW_MOUNTED, Arguments.createMap())
     }
   }
 
   override fun detachWebView() {
+    sendEvent(RNVisitableViewEvent.WEB_VIEW_UNMOUNTED, Arguments.createMap())
```

Comment:
Sent too late on Android. iOS has willDisappear callback and it's working properly for now. We can probably handle that event in a return statement in `useEffect` but maybe it will be more convenient to introduce another prop? I'm afraid about potential race condition though.